README.md

```diff
@@ -3,6 +3,22 @@
 This is the home of Tabzilla, the universal tab you can add to your website. Below
 follows simple instructions on how to get, and use Tabzilla.
 
+## Upgrading to the new Tabzilla
+
+Before you switch to the new Tabzilla, there are a couple of caveats you need to be
+aware of.
+
+- Tabzilla no longer needs any fonts bundled with it so, we are no longer including
+fonts in the CSS.
```

Comment:
Not sure I understand the question, I was just proposing a different wording? :smile: